src/hooks/reactQuery/wizards/useMyWizards.ts

```diff
@@ -206,3 +225,61 @@ export const useUpdateWizard = () => {
     },
   });
 };
+
+export const useUploadCustomImage = () => {
+  const wallet = useWallet();
+  return useMutation({
+    mutationFn: async ({
+      fileName,
+      base64Image,
+    }: UseUploadCustomImageProps) => {
+      if (wallet === undefined) {
+        throw Error("User not logged in");
+      }
+
+      const imageIdResponse: _SERVICE = await wallet.getCanisterActor(
+        elnaImagesCanisterId,
+        elnaImagesIdlFactory,
+        false
+      );
+      const response = imageIdResponse.add_asset({
+        asset: base64Image,
+        owner: Principal.fromText(wallet.principalId),
+        file_name: fileName,
+      },[])
+      return response;
+    },
+    onSuccess: (fileName) => {
+      queryClient.invalidateQueries({ queryKey: [QUERY_KEYS.AVATAR_IMAGE, fileName] });
+    },
+    onError: error => {
+      console.error(error);
+      toast.error(error.message);
+    },
+  });
+};
+
+export const useDeleteCustomImage = () => {
+  const wallet = useWallet();
+  return useMutation({
+    mutationFn: async (fileName: string) => {
+      if (wallet === undefined) {
+        throw Error("User not logged in");
+      }
+
+      const imageIdResponse: _SERVICE = await wallet.getCanisterActor(
+        elnaImagesCanisterId,
+        elnaImagesIdlFactory,
+        false
+      );
+      const response = imageIdResponse.delete_asset(fileName);
+      return response;
+    },
+    onSuccess: () => {
+    },
+    onError: error => {
+      console.error(error);
+      toast.error(error.message);
+    },
+  });
+};
```

Comment:
this is related to images right, it would make sense to have it in `src/hooks/reactQuery/useElnaImages.ts`

Reply:
Yes, moving these